_posts/2017-05-04-install-party.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: "Install Party + Social"
+slug: install-party
+text: 'tomesh monthly install party and social'
```

Comment:
This shows up in the description so maybe something a little differentiated from the title?

"Bring your equipment to set up a mesh node" (or something?)